projects/experimental/components/badge/badge.style.less

```diff
@@ -0,0 +1,461 @@
+@import 'taiga-ui-local';
+
+:host {
+    .transition(background-color);
+
+    --left-content-color: currentColor;
+    position: relative;
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
+    box-sizing: border-box;
+    color: var(--tui-base-01);
```

Comment:
Is it base-01 or text-01-night? Should it get inversed in night mode?

Reply:
seems like base-01 (with inversion) looks better